query-languages/m/itemexpression-from.md

```diff
@@ -5,50 +5,7 @@ ms.date: 10/7/2022
 ---
 # ItemExpression.From
 
-## Syntax
-
-<pre>
-ItemExpression.From(<b>function</b> as function) as record
-</pre>
-
 ## About
 
-Returns the abstract syntax tree (AST) for the body of `function`, normalized into an *item expression*:
-
-- The function must be a 1-argument lambda.
-- All references to the function parameter are replaced with [ItemExpression.Item](itemexpression-item.md).
-- The AST will be simplified to contain only nodes of the kinds:
-  - `Constant`
-  - `Invocation`
-  - `Unary`
-  - `Binary`
-  - `If`
-  - `FieldAccess`
-  - `NotImplemented`
-
-An error is raised if an item expression AST cannot be returned for the body of `function`.
-
-## Example 1
-
-Returns the AST for the body of the function `each _ <> null`
-
-**Usage**
-
-```powerquery-m
-ItemExpression.From(each _ <> null)
-```
-
-**Output**
+This identifier is an alias for [RowExpression.From](rowexpression-from.md).
```

Comment:
Implementation-wise, it's actually the other way around.

Is anyone actually using these functions externally?

Reply:
> Is anyone actually using these functions externally?

I believe a small group of people do. I've seen forum/GitHub content touching on it ([example1]( https://community.powerbi.com/t5/Power-Query/Query-folding-Table-View-handler-function-to-filter-rows-add/m-p/3155040/highlight/true#M101208), [example2](https://github.com/CurtHagenlocher/CSQL/blob/master/CSQL.pq)). Then there's me, of course. :-)

Reply:
> Implementation-wise, it's actually the other way around.

Would you like me to reverse this change so that the aliases point the other way (e.g. `RowExpression.From` is an alias for `ItemExpression.From`)? 

If I did this, the trio of methods would be `ItemExpression.From` (with `RowExpression.From` alias), `ItemExpression.Item` (with `RowExpression.Row` alias) and then `RowExpression.Column` (with no alias)—so two `ItemExpression.*` methods and then one `ItemExpression.*` method. (Should an `ItemExpression.Column` be added?)

On the other hand, if we leave the change the way it is, the trio is `Row.Expression.From` (with `ItemExpression.From` alias), `RowExpression.Row` (with `ItemExpression.Item` alias) and `RowExpression.Column` (with no alias)—so all three are `RowExpression.*` methods.

Reply:
What if we just left the documentation for each function in place, but added a line about equivalency to each function's About. For example:

For ItemExpression.From:
ItemExpression.From is equivalent to RowExpression.From.

And for RowExpression.From:
RowExpression.From is equivalent to ItemExpression.From.

Reply:
Or we could say "identical" instead of "equivalent".

Reply:
Sounds like a plan.